deploy/olm-catalog/performance-addon-operator/0.0.1/performance-addon-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -29,12 +30,13 @@ metadata:
               "node-role.kubernetes.io/performance": ""
             },
             "realTimeKernel": {
-              "repoURL": "http://download-node-02.eng.bos.redhat.com/rhel-8/nightly/RHEL-8/latest-RHEL-8.1.1/compose/RT/x86_64/os"
+              "enabled": true
             }
           }
         }
       ]
     capabilities: Basic Install
+    containerImage: REPLACE_IMAGE
```

Comment:
where is this suddenly coming from... :thinking:

Reply:
I think @davidvossel said he will check from where does it appear.

Reply:
Let's blame @fromanirh ;)
https://github.com/openshift-kni/performance-addon-operators/commit/be0dc8dcc1994b08c924e6a881b1a2964c64d451
So all good :)
But we should add a CI check if the CSV is updated when running `make generate-latest-dev-csv`

Reply:
yep, this is intentional.

Reply:
yep will add the check